src/code/z_message.c

```diff
@@ -67,11 +67,11 @@ void func_80147520(void) {
 void func_80147564(PlayState* play) {
     MessageContext* msgCtx = &play->msgCtx;
 
-    msgCtx->unk1204A[0] = 0xBD;
-    msgCtx->unk1204A[1] = 0xB8;
-    msgCtx->unk1204A[2] = 0xB3;
-    msgCtx->unk1204A[3] = 0xAE;
-    msgCtx->unk1204A[4] = 0xA9;
+    msgCtx->ocarinaButtonsPosY[0] = 0xBD;
+    msgCtx->ocarinaButtonsPosY[1] = 0xB8;
+    msgCtx->ocarinaButtonsPosY[2] = 0xB3;
```

Comment:
```suggestion
    msgCtx->ocarinaButtonsPosY[OCARINA_BTN_A] = 0xBD;
    msgCtx->ocarinaButtonsPosY[OCARINA_BTN_C_DOWN] = 0xB8;
    msgCtx->ocarinaButtonsPosY[2] = 0xB3;
```
And for the rest. Can also probably be decimal, although we can do a decimal cleanup later